packages/resolvers/default/src/DefaultResolver.js

```diff
@@ -17,7 +17,7 @@ import builtins from './builtins';
 export default new Resolver({
   async resolve({dependency, options}) {
     const resolved = await new NodeResolver({
-      extensions: ['ts', 'tsx', 'js', 'json', 'css', 'styl'],
+      extensions: ['ts', 'tsx', 'js', 'json', 'css', 'styl', 'less'],
```

Comment:
I think we talked about not adding to this default extensions list.

Reply:
I agree that this is a bit icky, but not doing so would break the case for 

```less
// vars.less
@red: #00dd00;

// main.less
@import "a";
body {
  color: @red;
}
```

But I may be able to make the resolvePathPlugin a bit smarter.

Reply:
We should probably fallback to the default less resolver

Reply:
This change actually does nothing semantically, because `NodeResolver` [not only prioritizes the parents' expression, it adds it even if it's not there](https://github.com/parcel-bundler/parcel/blob/v2/packages/resolvers/default/src/DefaultResolver.js#L114-L118), so cases like `@import "~library/styles";` will still work.

Reply:
Ow great didn't know about that haven't read through the resolver in a lil while